### PR TITLE
Update bfrt_dpdk_sample_config.conf

### PR DESCRIPTION
**Pull request by swaroopsarma**

### CODE REVIEW

**Review by vsureshkumarp — APPROVED** [review]
Changes looks good to me.

**Review by asunkad — APPROVED** [review]

File: bf_switchd/bfrt_dpdk_sample_config.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@
         {
             "id": "asic-0",
             "chip_family": "dpdk",
-            "instance": 0,
+            "instance": 0
         }
     ],
     "instance": 0,
```
Comment (asunkad) [inline bf_switchd/bfrt_dpdk_sample_config.conf:9]: this field can be delete, we are not parsing this field in code

**Review by vsureshkumarp — APPROVED** [review]
Changes looks good to me

**Review by asunkad — COMMENTED** [review]
LGTM

**Pull request merged**